eo-runtime/src/test/java/JulToSlf4jExtension.java

```diff
@@ -0,0 +1,24 @@
+import org.junit.jupiter.api.extension.BeforeAllCallback;
```

Comment:
@marceloamadeu see other classes, they all have licenses at the top of the file

Reply:
@yegor256 licenses were added.